torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1052,6 +1052,19 @@ XLANativeFunctions::convolution_backward_overrideable(
                      : at::Tensor());
 }
 
+at::Tensor XLANativeFunctions::copy(const at::Tensor& self,
+                                    const at::Tensor& src, bool non_blocking) {
+  TORCH_LAZY_FN_COUNTER("xla::");
+  return _copy_from(src, self, non_blocking);
+}
+
+at::Tensor& XLANativeFunctions::copy_(at::Tensor& self, const at::Tensor& src,
+                                      bool non_blocking) {
+  TORCH_LAZY_FN_COUNTER("xla::");
+  at::Tensor result = _copy_from(self, src, non_blocking);
+  return result;
```

Comment:
Would it be better to?
```
_copy_from(self, src, non_blocking);
return self;
```

Reply:
Okay, I noticed that `test_copy_noncontig_xla` test was failing with:
```
AssertionError: Tensor-likes are not close!
```
Looking back at this, it should've been `_copy_from(src, self, non_blocking)`, not `_copy_from(self, src, non_blocking)`. After changing this, the `test_copy_noncontig_xla` test is now passing locally. I'll wait for the CI to verify. Now this op looks like:
```
at::Tensor& XLANativeFunctions::copy_(at::Tensor& self, const at::Tensor& src,
                                      bool non_blocking) {
  TORCH_LAZY_FN_COUNTER("xla::");
  _copy_from(src, self, non_blocking);
  return self;
}
```

Reply:
Thanks for the update. Can you leave a comment on the commit log to suggest why both copy and copy_ are lowered? It could be simple as we don't want to use the auto-generated copy_ lowering.

Reply:
Sounds good, updated.